Molyktech/Property-Pro-Lite
# #166872280 Endpoint for  Get all property advert

#### What does this PR do?
add  endpoint to Get all property advert

#### Description of Task to be completed?
have the  endpoint working:
`` GET 'http://localhost:3000/api/v1/property'``

#### How should this be manually tested?
``clone the repo``
``cd into the directory Property-Pro-Lite``
``run npm install``
``run npm start``
``using POSTMAN, test by making a GET request to 'http://localhost:3000/api/v1/property'``

make sure the header is set to:
 `` key: Content-Type ``
``value: application/json ``


#### Any background context you want to provide?
NIL

#### What are the relevant pivotal tracker stories?
[#166872280](https://www.pivotaltracker.com/story/show/166872280)

#### Screenshots (if appropriate)
NIL

#### Questions:
NIL
